### PR TITLE
chore: remove unneeded dependencies

### PR DESCRIPTION
**Pull request by ferferga**
Removes unneeded dependencies, move comlink-loader to development and upgrade the ones that have pending dependabot updates.

Ignore the branch name, I intended to bring Nuxt bridge to the project but it's really unstable and everything was borked, so it seems sensible to wait until things stabilize for Nuxt 3.

### CODE REVIEW

**Comment by ferferga** [comment]
Seems jest didn't enjoy being updated

**Comment by ferferga** [comment]
@jellyfin-bot rebase

File: .eslintrc.js [hunk 1]
```diff
@@ -116,7 +116,10 @@ module.exports = {
             }
           }
         ],
-        'vue/multiline-html-element-content-newline': 'error'
+        'vue/multiline-html-element-content-newline': 'error',
+        'vue/multi-word-component-names': 'off',
+        // This rule gives false positives with asyncData
+        'vue/no-dupe-keys': 'off'
```
Comment (ThibaultNocchi) [inline .eslintrc.js:122]: Just tried building and testing with these removed (ie the rules being applied and not offed) and it worked. Is there a reason for removing them that I missed?

Comment (ferferga) [reply .eslintrc.js:122]: Build works, it's linter that complains :)

**Comment by ThibaultNocchi** [comment]
@jellyfin-bot rebase

**Review by ThibaultNocchi — APPROVED** [review]

**Pull request merged**